src/Compilers/CSharp/CSharpAnalyzerDriver/CSharpDeclarationComputer.cs

```diff
@@ -117,6 +115,20 @@ private static bool InvalidLevel(int? level)
                             return;
                         }
 
+                        goto case SyntaxKind.ClassDeclaration;
+                    }
+                case SyntaxKind.RecordStructDeclaration:
+                    {
+                        if (associatedSymbol is IMethodSymbol ctor)
+                        {
```

Comment:
:memo: I didn't manage to cover this `if` and couldn't find coverage for above `if` either.

Reply:
> I didn't manage to cover this if and couldn't find coverage for above if either.

Really? I acn easily hit code paths in the previous case by running existing test in src\Compilers\CSharp\Test\Semantic\Semantics\RecordTests.cs

---
In reply to: [604290986](https://github.com/dotnet/roslyn/pull/52256#discussion_r604290986) [](ancestors = 604290986)

Reply:
Thanks. You're right. Both `if` blocks are visited by the `AnalyzerActions_XX` tests in `RecordTests` and `RecordStructTests`.

---
In reply to: [605097075](https://github.com/dotnet/roslyn/pull/52256#discussion_r605097075) [](ancestors = 605097075,604290986)